CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Added - 24.5.0
 
+- SC-5287 Fixed OAuth2 rostering
```

Comment:
this should be added as new version above https://github.com/hpi-schul-cloud/schulcloud-server/pull/1530/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR30